gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -361,7 +361,7 @@ static void
 record_statistics (GstTensorFilterPrivate * priv)
 {
   gint64 end_time = g_get_real_time ();
-  gint64 *latency = g_new (gint64, 1);
+  gint64 *latency = g_new0 (gint64, 1);
```

Comment:
This is not required. Line 367 writes to *latency anyway.

Reply:
Thanks, you're right! I removed the change.